src/wp-includes/theme.php

```diff
@@ -3654,6 +3662,14 @@ function wp_customize_url( $stylesheet = '' ) {
 function wp_customize_support_script() {
 	$admin_origin = parse_url( admin_url() );
 	$home_origin  = parse_url( home_url() );
+	
+	if(!isset($admin_origin['host'])){
+		$admin_origin['host'] = '';
+	}
+	if(!isset($home_origin['host'])){
```

Comment:
```suggestion
	if ( ! isset( $home_origin['host'] ) ) {
```
Coding standards fixes.